internal/api/group_test.go

```diff
@@ -351,13 +351,43 @@ func TestDeleteGroup(t *testing.T) {
 	})
 	require.NoError(t, err)
 
+	installation1, err := client.CreateInstallation(&model.CreateInstallationRequest{
+		OwnerID:  "owner",
+		Version:  "version",
+		DNS:      "dns.example.com",
+		Affinity: model.InstallationAffinityIsolated,
+	})
```

Comment:
nit: in most places we have an empty newline after closing a stanza; I would personally like to keep doing that, here and in a few other places.